### ISSUE TITLE
nvda doesn't read the items in a chm treeview

### ISSUE THREAD

**Issue by vlad-a-c** [nvaccess/nvda#11814]
#### Did you try to run the COM registry fixing tool in NVDA menu / tools?
hello.
i'm not sure exactly why is this happening but since this version on, 2020.4, nvda doesn't read the items in a chm treeview. it says unknown.

**Comment by Adriani90** [comment]
Closing as duplicate of #11797
